remote-api/src/test/java/org/alfresco/repo/web/scripts/rule/RuleServiceTest.java

```diff
@@ -1,38 +1,40 @@
-/*
```

Comment:
There is a ton of modifications shown, but it's mostly refactoring, moving methods so that they would go in order setUp -> shutdown -> tests -> check methods -> additional methods. With JSON building methods extracted to RuleJsonUtil.
The only real changes here are addition of 2 tests:
testCreatePrivateActionRule
testUpdatePrivateActionRule

addition of custom createRule method, allowing to pass JSONObject consumer and expected status,
addition of verifyPrivateActionFailureResponse method validating error response.